src/components/Modals/Onboarding/OnboardingBasicSingleDropdown.vue

```diff
@@ -9,24 +9,28 @@
         class="onboarding-dropdown-placeholder college-major-minor-wrapper"
         @click="showHideDropdown()"
       >
-        <div
+        <input
+          type="text"
           class="onboarding-dropdown-placeholder college-major-minor-placeholder"
-          :style="{ color: placeholderColor }"
-        >
-          {{ availableChoices[choice] || 'Select one' }}
-        </div>
+          :style="{ color: placeholderColor, border: 'none', outline: 'none' }"
+          :placeholder="prevQuery === '' ? 'Select one' : prevQuery"
```

Comment:
When I click + another major/minor, the Select One placeholder doesn't seem to appear and we have a blank box.
![Screen Shot 2021-05-15 at 12 44 55 PM](https://user-images.githubusercontent.com/54298311/118371613-5b29c100-b57b-11eb-81b5-e32c943e0af5.png)